### PR TITLE
OCaml Workshop 2022

### PR DESCRIPTION
**Pull request by cuihtlauac**
Update page with presentations title, authors and pre-print links

### CODE REVIEW

File: data/workshops/2022.md [hunk 1]
```diff
@@ -9,13 +9,91 @@ important_dates:
     info: Author notification
   - date: 2022-09-16
     info: OCaml Workshop
-presentations: []
+presentations:
+  - title: "Composing Schedulers using Effect Handlers"
```
Comment (christinerose) [inline data/workshops/2022.md]:
```suggestion
  - title: "Composing Schedulers Using Effect Handlers"
```

File: data/workshops/2022.md [hunk 2]
```diff
@@ -9,13 +9,91 @@ important_dates:
...
+    authors:
+      - Deepali Ande
+      - KC Sivaramakrishnan
+    link: https://kcsrk.info/papers/compose_ocaml22.pdf
+  - title: "Continuous Monitoring of OCaml Applications using Runtime Events"
```
Comment (christinerose) [inline data/workshops/2022.md]:
```suggestion
  - title: "Continuous Monitoring of OCaml Applications Using Runtime Events"
```

File: data/workshops/2022.md [hunk 3]
```diff
@@ -9,13 +9,91 @@ important_dates:
...
+    authors:
+      - Sadiq Jaffer
+      - Patrick Ferris
+    link: https://github.com/patricoferris/runtime-events-demo/blob/main/abstract.pdf
+  - title: "Copying opam switches - it should Just Work™"
```
Comment (christinerose) [inline data/workshops/2022.md]:
```suggestion
  - title: "Copying opam Switches - It Should Just Work™"
```

File: data/workshops/2022.md [hunk 4]
```diff
@@ -9,13 +9,91 @@ important_dates:
...
+    authors:
+      - David Allsopp
+  - title: "Efficient “out of heap” pointers for multicore OCaml"
```
Comment (christinerose) [inline data/workshops/2022.md]:
```suggestion
  - title: "Efficient “Out of Heap” Pointers for Multicore OCaml"
```

File: data/workshops/2022.md [hunk 5]
```diff
@@ -9,13 +9,91 @@ important_dates:
...
+    authors:
+      - Guillaume Munch-Maccagnoni
+    link: https://guillaume.munch.name/files/large-pages-ocamlworkshop.pdf
+  - title: "Highest-performance Stream Processing"
```
Comment (christinerose) [inline data/workshops/2022.md]:
```suggestion
  - title: "Highest-Performance Stream Processing"
```

File: data/workshops/2022.md [hunk 6]
```diff
@@ -9,13 +9,91 @@ important_dates:
...
+    authors:
+      - Oleg Kiselyov
+      - Tomoaki Kobayashi
+      - Aggelos Biboudis
+      - Nick Palladinos
+    link: https://okmij.org/ftp/meta-programming/strymonasv2-intro.pdf
+  - title: "Homogeneous builds with OBuilder and OCaml"
```
Comment (christinerose) [inline data/workshops/2022.md]:
```suggestion
  - title: "Homogeneous Builds with OBuilder and OCaml"
```

File: data/workshops/2022.md [hunk 7]
```diff
@@ -9,13 +9,91 @@ important_dates:
...
+      - Kate Deplaix
+    link: https://github.com/tmcgilchrist/ocaml-2022-submission/blob/master/ocurrent.pdf
+  - title: "Introducing the Bindoj library, a datatype-centric generative programming library for real-world programming in OCaml"
+    authors:
+      - Haochen M. Kotoi-Xie
+      - Hirotetsu Hongo
+      - Yuta Sato
+      - Shinya Yamaguchi
+    link: https://icfp22.sigplan.org/details/ocaml-2022-papers/14/Introducing-the-Bindoj-library-a-datatype-centric-generative-programming-library-for
+  - title: "Memo: an incremental computation library that powers Dune"
```
Comment (christinerose) [inline data/workshops/2022.md]:
```suggestion
  - title: "Memo: An Incremental Computation Library That Powers Dune"
```

File: data/workshops/2022.md [hunk 8]
```diff
@@ -9,13 +9,91 @@ important_dates:
...
+  - title: "OCamello: A Course and Summer School with Learn-OCaml"
+    authors:
+      - Roberto Blanco
+      - Ricardo J. Rodríguez
+    link: https://icfp22.sigplan.org/details/ocaml-2022-papers/1/OCamello-A-Course-and-Summer-School-with-Learn-OCaml
+  - title: "OCaml 5.0 - Concurrent and Parallel programming for OCaml"
+    authors:
+      - KC Sivaramakrishnan
+    link: https://icfp22.sigplan.org/details/ocaml-2022-papers/16/OCaml-5-0-Concurrent-and-Parallel-programming-for-OCaml
+  - title: "Stack allocation for OCaml"
```
Comment (christinerose) [inline data/workshops/2022.md]:
```suggestion
  - title: "Stack Allocation for OCaml"
```

File: data/workshops/2022.md [hunk 9]
```diff
@@ -9,13 +9,91 @@ important_dates:
...
+    authors:
+      - Stephen Dolan
+      - Leo White
+    link: http://stedolan.net/talks/ocaml22
+  - title: "Supporting a decade of opam"
+    authors:
+      - David Allsopp- Raja Boujbel- Kate Deplaix- Louis Gesbert
+  - title: "Supporting FLAT concepts in Learn-OCaml: seeing is believing, programming is understanding"
```
Comment (christinerose) [inline data/workshops/2022.md]:
```suggestion
  - title: "Supporting FLAT Concepts in Learn-OCaml: Seeing is Believing; Programming is Understanding"
```

File: data/workshops/2022.md [hunk 10]
```diff
@@ -40,28 +118,11 @@ program_committee:
     affiliation: FAMAF, UNC and Manas.Tech, Argentina
 ---
 
-## Scope
+OCaml Workshop 2022 was co-located with ICFP 2022, in Ljubljana, Slovenia
```
Comment (christinerose) [inline data/workshops/2022.md]: What does co-located mean in this instance, as only one location is mentioned?

File: data/workshops/2022.md [hunk 11]
```diff
@@ -40,28 +118,11 @@ program_committee:
...
-
-The main presentation format is a workshop talk, traditionally around 20 minutes in length, plus question time, but we also have a poster session during the workshop - this allows to present more diverse work, and gives time for discussion. The program committee will decide which presentations should be delivered as posters or talks.
-
-## Submission
-
-To submit a presentation, please register a description of the talk (about 2 pages long) at [https://ocaml2022.hotcrp.com/](https://ocaml2022.hotcrp.com/) providing a clear statement of what will be provided by the presentation: the problems that are addressed, the solutions or methods that are proposed.
-
-LaTeX-produced PDFs are a common and welcome submission format. For accessibility purposes, we ask PDF submitters to also provide the sources of their submission in a textual format, such as .tex sources. Reviewers may read either the submitted PDF or the text version.
-
-## ML family workshop
```
Comment (christinerose) [inline data/workshops/2022.md:65]: ML Family Workshop
(capitalized)

File: data/workshops/2022.md [hunk 12]
```diff
@@ -9,13 +9,91 @@ important_dates:
...
+    authors:
+      - Tim McGilchrist
+      - David Allsopp
+      - Patrick Ferris
+      - Antonin Décimo
+      - Thomas Leonard
+      - Anil Madhavapeddy
+      - Kate Deplaix
+    link: https://github.com/tmcgilchrist/ocaml-2022-submission/blob/master/ocurrent.pdf
+  - title: "Introducing the Bindoj library, a datatype-centric generative programming library for real-world programming in OCaml"
```
Comment (christinerose) [inline data/workshops/2022.md]:
```suggestion
  - title: "Introducing the Bindoj Library, a Datatype-Centric Generative Programming Library for Real-World Programming in OCaml"
```

**Review by christinerose — CHANGES_REQUESTED** [review]
Capitalization suggestions in titles

File: data/workshops/2022.md (hunk 11 above)
Comment (cuihtlauac) [reply data/workshops/2022.md:65]: Many thanks for all your comments. I've capitalized all the titles, hopefully. This latter text was deleted

File: data/workshops/2022.md (hunk 10 above)
Comment (cuihtlauac) [reply data/workshops/2022.md]: Replaced "was co-located with" by "took place during"

**Review by patricoferris — APPROVED** [review]
Thanks!

We could link to https://watch.ocaml.org/video-channels/ocaml2022/videos even thought it only has one video :))

**Comment by cuihtlauac** [comment]
> Thanks!
> 
> We could link to https://watch.ocaml.org/video-channels/ocaml2022/videos even thought it only has one video :))

Done

**Review by tmattio — APPROVED** [review]
LGTM, thanks @cuihtlauac!

While testing I noticed that the logic to display the videos on the workshop page is broken (the videos are never displayed), but it's unrelated to the current PR.

**Comment by avsm** [comment]
We can mirror these videos from the YouTube channel to watch.ocaml.org, which is my preferred mechanism to link to them from the OCaml site.  Any volunteers want to have a go with PeerTube (let me know and I can create an account).  If not, I'll do that shortly.

**Comment by cuihtlauac** [comment]
> We can mirror these videos from the YouTube channel to watch.ocaml.org, which is my preferred mechanism to link to them from the OCaml site. Any volunteers want to have a go with PeerTube (let me know and I can create an account). If not, I'll do that shortly.

I volunteer.

**Review by christinerose — APPROVED** [review]

**Comment by tmattio** [comment]
Thanks @cuihtlauac! This looks good as-is, so merging now and we can add the videos in a follow-up PR.

**Pull request merged**